src/System.Reflection.Metadata/tests/Metadata/Decoding/OpaqueHandleTypeProvider.cs

```diff
@@ -10,19 +10,19 @@ namespace System.Reflection.Metadata.Decoding.Tests
     // Used for test signature blobs that we didn't get from an actual metadata reader.
     internal class OpaqueTokenTypeProvider : DisassemblingTypeProvider
     {
-        public override string GetTypeFromHandle(MetadataReader reader, EntityHandle handle)
+        public override string GetTypeFromHandle(MetadataReader reader, DisassemblingGenericContext genericContext, EntityHandle handle)
         {
             return MetadataTokens.GetToken(handle).ToString("X");
```

Comment:
>MetadataTokens.GetToken(handle).ToString("X") [](start = 19, length = 45)

Extract to a static method and call the method from these 3 methods. It's just test, but it's a good sample for usage and i think calling GetTypeFromHandle with genericContext: null is weird.